lib/render/site-award.php

```diff
@@ -66,53 +66,61 @@ function RenderSiteAwards($userAwards): void
 function RenderAwardGroup($awards, $title): void
 {
     $numItems = is_countable($awards) ? count($awards) : 0;
-    if ($numItems == 0) {
+    $numHidden = 0;
+    foreach ($awards as $award) {
+        if ($award['DisplayOrder'] != -1) break;
+        $numHidden++;
```

Comment:
Any negative value can hide the items.

Also, the break here doesn't really provide any benefits. Recommend inverting the logic:
```
if ($award['DisplayOrder'] < 0) {
    $numHidden++;
}
```

Reply:
The break was just a minor optimization since `DisplayOrder` is in increasing order. But indeed, the latter way is cleaner.